src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -217,15 +220,19 @@ public ArmSubsystem() {
 
   //Returns sim encoder position (No offset) if in simulation, the actual position otherwise
   public double getBaseAngle() {
```

Comment:
Check references to this method, and check whether they should be switched to getChooChooAngle